lib/GADS/Users.pm

```diff
@@ -101,7 +107,9 @@ sub user_summary_rs
 
 sub _build_all
 {   my $self = shift;
-    my @users = $self->user_summary_rs->all;
+    my @users;
+    @users = $self->user_summary_rs->all unless $self->person_filter;
+    @users = $self->user_summary_rs->search($self->person_filter)->all if $self->person_filter;
```

Comment:
This could be better written as:
```
my $users = $self->user_summary_rs;
$users = $users->search($self->person_filter) if $self->person_filter;
[$users->all];
```